Detections/VMwareCarbonBlack/CriticalThreatDetected.yaml

```diff
@@ -17,13 +17,13 @@ relevantTechniques:
   - T1210
 query: |
 
-  let threshold = 8;
-  CarbonBlackNotifications_CL
-  | where threatHunterInfo_score_d >= threshold
-  | extend eventTime = datetime(1970-01-01) + tolong(threatHunterInfo_time_d/1000) * 1sec
-  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), count() by eventTime, Threat_Name = threatHunterInfo_reportName_s, Device_Name = deviceInfo_deviceName_s,  Internal_IP = deviceInfo_internalIpAddress_s, External_IP = deviceInfo_externalIpAddress_s, Threat_Score = threatHunterInfo_score_d
-  | project-away count_
-  | extend timestamp = StartTime, HostCustomEntity = Device_Name, IPCustomEntity = Internal_IP
+let threshold = 8;
+CarbonBlackNotifications_CL
+| where threatInfo_score_d >= threshold
+| extend eventTime = datetime(1970-01-01) + tolong(threatInfo_time_d / 1000) * 1sec
+| summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), count() by eventTime, Threat_Name = threatInfo_summary_s, Device_Name = deviceInfo_deviceName_s, Internal_IP = deviceInfo_internalIpAddress_s, External_IP = deviceInfo_externalIpAddress_s, Threat_Score = threatInfo_score_d
+| project-away count_
+| extend timestamp = StartTime, HostCustomEntity = Device_Name, IPCustomEntity = Internal_IP
 entityMappings:
   - entityType: Host
     fieldMappings:
```

Comment:
Looks like you may have an old version of master or you accidentally removed the version and kind properties from the bottom of the YAML - https://github.com/Azure/Azure-Sentinel/blob/master/Detections/VMwareCarbonBlack/CriticalThreatDetected.yaml.  You will need to increment it to 1.1.0 otherwise the version change validation check will fail the build.

```yaml
version: 1.0.0
kind: Scheduled
```